cmd/local-gadget/profile/block-io.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package profile
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/cilium/ebpf"
+	"github.com/spf13/cobra"
+
+	commonprofile "github.com/inspektor-gadget/inspektor-gadget/cmd/common/profile"
+	"github.com/inspektor-gadget/inspektor-gadget/cmd/local-gadget/utils"
+	containerutils "github.com/inspektor-gadget/inspektor-gadget/pkg/container-utils"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadget-collection/gadgets/profile"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/profile/block-io/tracer"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/profile/block-io/types"
```

Comment:
nit: we are using notation <gadget>Types and <gadget>Tracer:

```suggestion
	bioTracer "github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/profile/block-io/tracer"
	bioTypes "github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/profile/block-io/types"
```